src/ui/DropdownFilter.css

```diff
@@ -0,0 +1,13 @@
+.dropdown-display {
+    float:right;
+    width:150px;
+    height:30px;
+    cursor:pointer;
+    border:solid 2px #DDD;
+}
+
+.caption-display {
+    font-family: -webkit-pictograph;
+    font-size: 19px;
+    height:30px;
+}
```

Comment:
Add styling for placeholder https://stackoverflow.com/questions/5805059/how-do-i-make-a-placeholder-for-a-select-box

Probably need to add a class ".placeholder-selected"

In Mendix moderl it is called the "Empty option caption"
https://docs.mendix.com/refguide/drop_down#general-properties